addons/web/static/src/views/list/list_renderer.scss

```diff
@@ -475,3 +475,7 @@
         margin: auto 0;
     }
 }
+
+.modal-footer .o_list_selection_box {
```

Comment:
Can you add a commit message to explain the issue to follow the guidelines? https://www.odoo.com/documentation/master/contributing/development/git_guidelines.html#commit-message-structure

It would help the reviewers but also the ones who will blame the code.

Reply:
Yes obviously :grimacing: sorry and thanks for the review

Reply:
No problem 😉 
Thanks! 😁